### PR TITLE
merge this

### PR DESCRIPTION
**Pull request by voxtex**

### CODE REVIEW

**Comment by awilliamson** [comment]
:+1:

File: main.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 var POLL_INTERVAL = 60; // how often to check the open PRs (in seconds)
 
-var config = require('./config.js');
+var config = require␀('./config.js');
```
Comment (mayli) [inline main.js]: What is the `␀` here :question:  :-1:

**Comment by hintss** [comment]
-1

**Comment by mszaro** [comment]
:-1: Breaks the bot

**Comment by rdancer** [comment]
:-1:

**Comment by thmas** [comment]
:-1:

**Comment by AgentME** [comment]
:-1:

**Comment by avdg** [comment]
:-1: Needs motivation

**Comment by dos1** [comment]
:-1:

**Comment by thmas** [comment]
:-1:

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting has begun.

To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must star this repo for your vote to count.**

A decision will be made after this PR has been open for **30** minutes, and at least **5** votes have been made.

*NOTE: the PR will be closed if any new commits are added after:* 52fd4fbb0c5a051e812def59205a2a8ca4c3ce64

**Comment by begizi** [comment]
:-1:

**Comment by JWesorick** [comment]
:-1:

**Comment by botwillacceptanything** [comment]
:-1:

**Comment by mappum** [comment]
:-1:

**Comment by AgentME** [comment]
Do votes before the bot's post count?
:-1:

**Comment by mappum** [comment]
@AgentME Nope. That could be changed though.

The only reason the bot's post is so late is because it was getting rate limited, and it might not be anymore.

**Comment by dos1** [comment]
Wow, that sucks.

**Comment by jvt** [comment]
:-1:

**Comment by botwillacceptanything** [comment]
#### :-1: The vote failed, this PR will now be closed

**Pull request closed without merging**